starter-code/index.js

```diff
@@ -1,11 +1,47 @@
 class SortedList {
-    constructor() {}
-    add(item) {}
-    get(pos) {}
-    max() {}
-    min() {}
-    average() {}
-    sum() {}
+    constructor(length,items) {
+      this.length=0;
+      this.items=[];
+    }
+    add(item) {
+      this.items.push(item);
+      this.length +=1;
+      return this.items.sort((a, b) => a - b);
+
+    }
+    get(pos) {
```

Comment:
Il manque le throw de l'erreur : 
 get(pos) {
    if (pos > this.length) {
      throw new Error("OutOfBounds");
    }
    return this.items[pos];
  }